INSTALL.md

```diff
@@ -0,0 +1,81 @@
+# Installation
+
+## Availability
+
+The sources for AMICI are accessible as
+- Source  [tarball](https://github.com/ICB-DCM/AMICI/tarball/master)
+- Source  [zip](https://github.com/ICB-DCM/AMICI/zipball/master)
+- GIT repository on  [github](https://github.com/ICB-DCM/AMICI)
+
+### Obtaining AMICI via the GIT versioning system
+In order to always stay up-to-date with the latest AMICI versions, simply pull it from our GIT repository and
+recompile it when a new release is available. For more information about GIT checkout their [website](http://git-scm.com/)
+
+The GIT repository can currently be found at https://github.com/ICB-DCM/AMICI and a direct clone is possible via
+
+    git clone https://github.com/ICB-DCM/AMICI.git AMICI
+
+## Installation
+
+If AMICI was downloaded as a zip, it needs to be unpacked in a convenient directory. If AMICI was obtained via cloning of the git repository, no further unpacking is necessary.
+
+### MATLAB
+
+To use AMICI from MATLAB, start MATLAB and add the AMICI/matlab direcory to the MATLAB path. To add all toolbox directories to the MATLAB path, execute the matlab script
+
+    installAMICI.m
+
+To store the installation for further MATLAB session, the path can be saved via
+
+    savepath
+
+For the compilation of .mex files, MATLAB needs to be configured with a working C compiler. The C compiler needs to be installed and configured via:
+
+    mex -setup c
+
+For a list of supported compilers we refer to the mathworks documentation: [mathworks.com](http://mathworks.com/support/compilers/R2017a/index.html)
+
+###  Python
+
+To use AMICI from Python, install the module using pip  
+
+    pip3 install amici
+    
+You can now import it as python module:
+
+    import amici
+
+### C++
+
+To use AMICI from C++, run the  
+
+    ./scripts/buildSundials.sh
+    ./scripts/buildSuitesparse.sh
+    ./scripts/buildAmici.sh
+    
+script to compile amici libary. The static library file can then be linked from 
+
+    ./build/libamici.a
```

Comment:
+
By default, AMICI exports its package configuration to the [CMake registry](https://cmake.org/cmake/help/v3.0/manual/cmake-packages.7.html#package-registry) from where it can easily be used by downstream CMake-based packages using `find_package(Amici)`.

Reply:
done